grafana/README.md

```diff
@@ -0,0 +1,24 @@
+# Grafana Dashboard
+
+You can find the Grafana dashboard in this directory that visualizes the KoP metrics.
+
+## Prerequisites
+
+The dashboard has been tested with the following software versions:
+
+* KoP >= 2.9.0
+* Grafana >= v8.4.3
+* Prometheus >= v2.0.0
+
+A Prometheus data source needs to be [added](https://prometheus.io/docs/visualization/grafana/#using) before installing the dashboard.
+
+## Installing the Dashboard
+
+In the Grafana UI complete the following steps:
```

Comment:
```suggestion
To install the Grafana Dashboard, follow these steps in the Grafana UI:
```
One question: which word is more accurate, GUI, console, or UI?

Reply:
Grafana provided a Web interface. Should we call it UI?